grin/src/Grin/ExtendedSyntax/Pretty.hs

```diff
@@ -146,7 +146,7 @@ instance Pretty Lit where
 instance Pretty BPat where
   pretty = \case
     VarPat v    -> pretty v
-    AsPat v val -> pretty v <> pretty '@' <> pretty '(' <> pretty val <> pretty ')'
+    AsPat v val -> pretty v <> pretty '@' <> pretty val
```

Comment:
Minor: Calling it 'val' is a bit misleading here.

Reply:
Renamed it to `pat`.